src/logic/operate.js

```diff
@@ -0,0 +1,24 @@
+import Big from "big.js";
+
+function operate(numberOne, numberTwo, operation) {
+  let result = 0;
+  if (operation === "+") {
+    result = Big(numberOne) + Big(numberTwo);
+  }
+  if (operation === "-") {
+    result = Big(numberOne) - Big(numberTwo);
+  }
+  if (operation === "x") {
+    result = Big(numberOne) * Big(numberTwo);
+  }
+  if (operation === "÷") {
+    result = Big(numberOne) / Big(numberTwo);
+  }
+  if (operation === "%") {
+    result = Big();
+  }
+
+  return result;
+}
```

Comment:
- [ ] operate should switch operation and use plus, minus, times, div and modifier. With the wrapping mentioned above your code should be like ⬇️ 

`result = num1.plus(num2)`